project/app/.eslintrc.yaml

```diff
@@ -1,4 +1,6 @@
 ---
 extends: availity/browser
 rules:
-  unicorn/prefer-query-selector: off
\ No newline at end of file
+  unicorn/prefer-query-selector: off
+env:
```

Comment:
we define the default `env` properties in `eslint-config-availity`. https://github.com/Availity/eslint-config-availity/blob/master/browser.js#L14 I don't believe we want this change cc @GoPro16

Reply:
Correct, the eslint config we use is for the browser. If someone wanted to use the eslint config for a node project they can just import the base config into their projects.

Reply:
@GoPro16  How to address [this](https://github.com/Availity/availity-starter-react/blob/cc7a3206fb200bb848e4b61d4f1a72dd98e52647/project/app/App.test.js#L9) then?

` 9:8  error  'global' is not defined                                   no-undef`

Reply:
If that is an issue this is something that should be submitted as a PR to `eslint-config-availity`

Reply:
I think we just need to add 

```
env: {
    "jest/globals": true
}
```

to the linter config for `availity/browser`

Reply:
Sounds appropriate.